bpf/lib/common.h

```diff
@@ -162,7 +162,7 @@ struct endpoint_key {
 		union v6addr	ip6;
 	};
 	__u8 family;
-	__u8 pad4;
+	__u8 key;
```

Comment:
Nice fit ;-)

Reply:
right might be worth mentioning. We also use endpoint_key as both a key and value in the map. If we need more specifiers in the key at some point we could split the key/value pair up.